lib/studies/reply/adaptive_nav.dart

```diff
@@ -260,7 +263,7 @@ class _NavigationRailHeader extends StatelessWidget {
                         ).animate(animation),
                         child: const Icon(
                           Icons.arrow_left,
-                          color: ReplyColors.blue50,
+                          color: ReplyColors.white50,
```

Comment:
double checking, these icon colors works in both light and dark theme?

Reply:
Yes this color is visible in both modes.